universal-application-tool-0.0.1/app/views/FileUploadStrategy.java

```diff
@@ -0,0 +1,192 @@
+package views;
+
+import static j2html.TagCreator.a;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.each;
+import static j2html.TagCreator.form;
+import static j2html.attributes.Attr.FORM;
+import static j2html.attributes.Attr.HREF;
+import static views.BaseHtmlView.makeCsrfTokenInputTag;
+import static views.BaseHtmlView.submitButton;
+
+import controllers.applicant.routes;
+import j2html.tags.ContainerTag;
+import j2html.tags.Tag;
+import java.util.Optional;
+import play.mvc.Http.HttpVerbs;
+import services.MessageKey;
+import services.applicant.question.ApplicantQuestion;
+import services.applicant.question.FileUploadQuestion;
+import views.applicant.ApplicantProgramBlockEditView.Params;
+import views.questiontypes.ApplicantQuestionRendererFactory;
+import views.questiontypes.ApplicantQuestionRendererParams;
+import views.questiontypes.FileUploadQuestionRenderer;
+import views.style.ApplicantStyles;
+import views.style.Styles;
+
+public abstract class FileUploadStrategy {
+
+  static final String IMAGES_AND_PDF = "image/*,.pdf";
```

Comment:
question (blocking): are these MIME types? 

```suggestion
  static final String MIME_TYPES_IMAGES_AND_PDF = "image/*,.pdf";
```

Reply:
Yes! Done.